packages/shared/nodemon.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Should nodemon be used in server? It's only for server at the moment.

Reply:
right no I need to remove this file